### PR TITLE
Creating a dedicated comms setup doc

### PR DESCRIPTION
**Pull request by CatGarab**

### CODE REVIEW

**Comment by CatGarab** [comment]
[OBC initial comms doc](https://trello.com/c/Eu2irWME/887-obc-initial-comms-doc)

File: docs/obc-docs/comms-setup.rst [hunk 1]
```diff
@@ -1,5 +1,365 @@
...
+See the appropriate `KubOS for the {OBC} > Working with the {OBC}` document for more information.
+
+Running Locally
+~~~~~~~~~~~~~~~
+
+If you are not using the Kubos SDK image and are instead running everything locally, you will need
+to manually set up a serial connection.
+
+Open your terminal application of choice (PuTTY, minicom, etc) and set up a connection with the
+following configuration values:
```
Comment (cantino) [inline docs/obc-docs/comms-setup.rst]: Maybe show an example of the Linux / OS X command line for this?

File: docs/obc-docs/comms-setup.rst [hunk 2]
```diff
@@ -1,5 +1,365 @@
...
+
+Using the SDK
+~~~~~~~~~~~~~
+
+If the target board is correctly connected to your host computer, you should see a `/dev/ttyUSB\*`
+device in your SDK instance.
+The VM is set up to automatically forward any FTDI cables that connect to a `/dev/FTDI` device for
+ease-of-use.
+
+The Vagrant image comes pre-packaged with a minicom serial connection configuration file.
```
Comment (cantino) [inline docs/obc-docs/comms-setup.rst]: Two colons.

File: docs/obc-docs/comms-setup.rst [hunk 3]
```diff
@@ -1,5 +1,365 @@
...
+You can connect with this configuration file using this command::
+
+    $ minicom kubos
+
+Logging In
+~~~~~~~~~~
+
+Once connected, you can log in using either a user that you've created, the default `kubos` user,
```
Comment (cantino) [inline docs/obc-docs/comms-setup.rst]: (I assume we document how to change these defaults somewhere?)

File: docs/obc-docs/comms-setup.rst [hunk 4]
```diff
@@ -1,5 +1,365 @@
...
+
+::
+
+    Welcome to Kubos Linux
+
+    Kubos login: kubos
+    Password: 
+    /home/kubos # 
+    
+test line
```
Comment (cantino) [inline docs/obc-docs/comms-setup.rst]: *test line*?

File: docs/obc-docs/comms-setup.rst [hunk 5]
```diff
@@ -1,5 +1,365 @@
...
+    department in order to be assigned an IP address.
+    
+If you are connecting your OBC directly to your host computer's ethernet port, you'll need to first
+determine and/or set the IP address of the host computer for that connection.
+
+Commands like ``ifconfig`` (or ``ipconfig`` for Windows users) should display the current address of
+the ethernet connection. Worth noting, it may be necessary to first connect the OBC to the host
+computer in order for the connection to appear in the command output.
+
+Please consult your relevant OS' documentation for instructions on setting a static IP address.
```
Comment (cantino) [inline docs/obc-docs/comms-setup.rst]: *OS's*

File: docs/obc-docs/comms-setup.rst [hunk 6]
```diff
@@ -1,5 +1,365 @@
...
+
+Configuring the OBC
+~~~~~~~~~~~~~~~~~~~
+
+Connect an ethernet cable from the OBC to either your computer or an open network port.
+
+Log into the OBC using the debug console and then edit ``/etc/network/interfaces``.
+Update the IP address field to be an address of your choosing.
+
+Once updated, run the following commands in order to make the OBC use the new address::
```
Comment (cantino) [inline docs/obc-docs/comms-setup.rst]: Are the double colons a thing?

File: docs/obc-docs/comms-setup.rst (hunk 4 above)
Comment (CatGarab) [reply docs/obc-docs/comms-setup.rst]: LOL, whoops :sweat_smile:

File: docs/obc-docs/comms-setup.rst (hunk 6 above)
Comment (CatGarab) [reply docs/obc-docs/comms-setup.rst]: Yes, it's a shortcut.

```
Here's some code:

::
    Formatted code blurb
```

Is the same as this:

```
Here's some code::

    Formatted code blurb
```

File: docs/obc-docs/comms-setup.rst (hunk 5 above)
Comment (CatGarab) [reply docs/obc-docs/comms-setup.rst]: I disagree. As far as I'm aware, it's optional to leave out the "s" if the possessive word already ends in an "s". My preference is to omit it.

File: docs/obc-docs/comms-setup.rst (hunk 3 above)
Comment (CatGarab) [reply docs/obc-docs/comms-setup.rst]: [Yup](https://docs.kubos.com/1.15.0/os-docs/using-kubos-linux.html#user-accounts)

File: docs/obc-docs/comms-setup.rst (hunk 5 above)
Comment (cantino) [reply docs/obc-docs/comms-setup.rst]: I've heard both sides argued and don't really care. :smile:

Comment (jacoffey3) [reply docs/obc-docs/comms-setup.rst]: I feel strongly about this, but I refuse to weigh in and will silently judge the person I think is wrong.

**Pull request merged**